packages/coachmark/test/coachmark.test.ts

```diff
@@ -14,7 +14,7 @@ import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
 
 import '@spectrum-web-components/coachmark/sp-coachmark.js';
 import { Coachmark } from '@spectrum-web-components/coachmark';
-import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import { testForLitDevWarnings } from '../../../test/testing-helpers';
```

Comment:
Can you try with the `.js` added back?

Reply:
I adjusted it locally but I'm still running into a test failure in `coachmark-popover.test.ts`